### DISCUSSION TITLE
drag the features out of the game

### DISCUSSION THREAD

**Discussion by username418** [Lailloken/Exile-UI#90] [Ideas & Feedback]
Let us drag the features out of the game, like the notepad and map-info are useful but could be dragged outside of the game on a second monitor or outside of a windowed game.

**Comment by Lailloken** [comment]
I'll look into it. It behaves as it currently does so that panels that are being resized don't clip outside the monitor-area (I need to see if there's a way to reliably check monitor-boundaries).
